solidity/contracts/KeepRandomBeaconOperator.sol

```diff
@@ -61,6 +61,10 @@ contract KeepRandomBeaconOperator is ReentrancyGuard {
     // TODO: replace with a secure authorization protocol (addressed in RFC 11).
     TokenStaking internal stakingContract;
 
+    // Captures the minimum stake when group selection starts. This is to ensure the
+    // same staking weight divisor is applied for all member candidates participating.
+    uint256 internal groupSelectionMinimumStake;
```

Comment:
What do you think about putting this value in `GroupSelection.Storage groupSelection`? 
We already have there `ticketSubmissionStartBlock` and `inProgress`